roles/installer/templates/statefulsets/postgres.yaml.j2

```diff
@@ -51,6 +51,12 @@ spec:
         - image: '{{ _postgres_image }}'
           imagePullPolicy: '{{ image_pull_policy }}'
           name: postgres
+{% if security_context_settings.postgres is defined %}
```

Comment:
The security_context_settings.postgres key doesn't exist in the AWX schema.  All I see is `security_context_settings`  [here](https://github.com/ansible/awx-operator/blob/devel/config/crd/bases/awx.ansible.com_awxs.yaml#L1791). 


If we wanted to have specific security context settings for postgres (different from task/web, etc.), then we would probably want to define it under `postgres.security_context_setting`.  We don't have a precedent for this in the awx-operator yet, but there are examples in the eda-server-operator [here](https://github.com/ansible/eda-server-operator/blob/main/config/crd/bases/eda.ansible.com_edas.yaml#L1248-L1254).

For this PR, it might make more sense to just use the top level security_context_settings value everywhere (including the postgres sts with this PR).  What do you think? Would that solve your use case?

Reply:
@rooftopcellist  i did the requested changes i went with having seperate security_context_settings for postgres instead using the same security_context_settings that is already declared.